Calibration/HcalCalibAlgos/test/test_hosignal_collision_cfg.py

```diff
@@ -0,0 +1,148 @@
+# Auto generated configuration file
+# using: 
+# Revision: 1.19 
+# Source: /local/reps/CMSSW/CMSSW/Configuration/Applications/python/ConfigBuilder.py,v 
+# with command line options: step3 -n 1000 --conditions auto:run2_data -s RAW2DIGI,L1Reco,RECO --datatier RECO --customise SLHCUpgradeSimulations/Configuration/postLS1Customs.customisePostLS1 --eventcontent RECO --magField AutoFromDBCurrent --no_exec --filein /store/data/Commissioning2015/MinimumBias/RAW/v1/000/239/754/00000/94E8E718-75DB-E411-9D0E-02163E0123CC.root --scenario pp
+import FWCore.ParameterSet.Config as cms
+
+process = cms.Process('ReRECO')
+
+# import of standard configurations
+process.load('Configuration.StandardSequences.Services_cff')
+process.load('SimGeneral.HepPDTESSource.pythiapdt_cfi')
+#process.load('FWCore.MessageService.MessageLogger_cfi')
+process.load('Configuration.EventContent.EventContent_cff')
+process.load('SimGeneral.MixingModule.mixNoPU_cfi')
+process.load('Configuration.StandardSequences.GeometryRecoDB_cff')
+process.load('Configuration.StandardSequences.MagneticField_AutoFromDBCurrent_cff')
+process.load('Configuration.StandardSequences.RawToDigi_Data_cff')
+process.load('Configuration.StandardSequences.L1Reco_cff')
+process.load('Configuration.StandardSequences.EndOfProcess_cff')
+process.load('Configuration.StandardSequences.FrontierConditions_GlobalTag_condDBv2_cff')
+
+process.MessageLogger = cms.Service("MessageLogger",  
+ cout = cms.untracked.PSet(  
+  default = cms.untracked.PSet( ## kill all messages in the log  
+ 
+   limit = cms.untracked.int32(0)  
+  ),  
+  FwkJob = cms.untracked.PSet( ## but FwkJob category - those unlimitted  
+ 
+   limit = cms.untracked.int32(-1)  
+  )  
+ ),  
+ categories = cms.untracked.vstring('FwkJob'), 
+ destinations = cms.untracked.vstring('cout')  
+)  
+ 
+
+process.maxEvents = cms.untracked.PSet(
+    input = cms.untracked.int32(-10)
+)
+
+# Input source
+process.source = cms.Source("PoolSource",
+    fileNames = cms.untracked.vstring(
+'/store/data/Run2015D/SingleMuon/RECO/PromptReco-v3/000/256/676/00000/2844234C-425F-E511-90F2-02163E014719.root'
+),                                
+    secondaryFileNames = cms.untracked.vstring()
+)
+
+process.options = cms.untracked.PSet(
+
+)
+
+# Production Info
+process.configurationMetadata = cms.untracked.PSet(
+    annotation = cms.untracked.string('step3 nevts:1000'),
+    name = cms.untracked.string('Applications'),
+    version = cms.untracked.string('$Revision: 1.19 $')
+)
+
+# Additional output definition
+
+# Other statements
+from Configuration.AlCa.GlobalTag_condDBv2 import GlobalTag
+process.GlobalTag = GlobalTag(process.GlobalTag, 'auto:run2_data', '')
```

Comment:
@bsunanda in 80X you can safe remove the import from  `Configuration.AlCa.GlobalTag_condDBv2` and use instead the standard one `Configuration.AlCa.GlobalTag`